proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -42,6 +42,18 @@ enum OrderBy {
   ORDER_BY_NAME = 3;
 }
 
+enum ElementType {
+  ELEMENT_TYPE_UNSPECIFIED = 0;
+  ELEMENT_TYPE_SERVICE = 1;
+  ELEMENT_TYPE_METHOD = 2;
+  ELEMENT_TYPE_MESSAGE = 3;
+  ELEMENT_TYPE_EXTENSION = 4;
+  ELEMENT_TYPE_FIELD = 5;
+  ELEMENT_TYPE_ONEOF = 6;
```

Comment:
Is there some kind of priority given to different element types? Surfacing oneof names alongside message names would concern me.